zipkin-autoconfigure/storage-elasticsearch-http/src/main/java/zipkin/autoconfigure/storage/elasticsearch/http/ZipkinElasticsearchHttpStorageProperties.java

```diff
@@ -38,6 +39,11 @@
   /** Number of replicas (redundancy factor) per index. Defaults to 1.` */
   private int indexReplicas = 1;
 
+  /** username and password fields for basic auth if configured */
+  private String basicAuthUserName;
```

Comment:
simplify to username and password and javadoc the properties individually as it helps with tooling.

Ex. `username used for basic auth. Needed when Shield or X-Pack security is enabled`